trymito.io/pages/spreadsheet-automation.tsx

```diff
@@ -0,0 +1,334 @@
+      </Head>
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+          <section className={classNames(titleStyles.title_card, pageStyles.background_card, spreadsheetAutomationStyles.flex_col_mobile_row_desktop)}>
+            <div className={spreadsheetAutomationStyles.title_text_container}>
+              <h1>
+                The No-Code Approach to Spreadsheet Automation 
+              </h1>
+              <p>
+                Automate your spreadsheets in hours, not months.
+              </p>
+              <CTAButtons variant={'download'} align='left'/>
+            </div>
+            <div className={classNames(spreadsheetAutomationStyles.hero_video_container)}>
+              <video autoPlay loop disablePictureInPicture playsInline webkit-playsinline="true" muted>
+                  <source src="/formula_writing.mp4" />
+              </video>
+            </div>
+          </section>
+
+          <section className={textImageSplitStyles.text_image_section_container}>
+            <div className={pageStyles.subsection}>
+              <div className={classNames(textImageSplitStyles.functionality_media, 'display-desktop-only-inline-block')}>
+                <Image src={'/code_snippet.png'} alt='Mito generated code' width={500} height={250} layout='responsive'/>
+              </div>
+              <div className={textImageSplitStyles.functionality_text}>
+                <h1>
+                  The easiest way to <span className='text-highlight'>write Python code</span>
+                </h1>
+                <p>
+                  Every edit made in the Mito spreadsheet is automatically converted to Python code. 
+                </p>
+                <p>
+                  Edit your data just like you do in Excel and receive production ready Python code without searching Pandas documentation or Stack Overflow.
+                </p>
+              </div>
+              <div className={classNames(textImageSplitStyles.functionality_media, 'display-mobile-only-block')}>
+                <Image src={'/code_snippet.png'} alt='Mito generated code!' width={500} height={250} layout='responsive'/>
+              </div>
+            </div>
+            <div className={pageStyles.subsection}>
+              <div className={textImageSplitStyles.functionality_text}>
+                <h1>
+                  <span className='text-highlight'>Explore</span> your data visually
+                </h1>
+                <p> 
+                  There&apos;s no replacement for scrolling through your data. Enter fullscreen mode and build intuition about your analysis.
+                </p>
+                <p>
+                  Use Mito&apos;s Excel-like interface to view CSV/Excel files and dataframes. And further explore your data by graphing, viewing summary stats, and creating pivot tables.
```

Comment:
Like on the main page, I could see these linking to other pages! E.g. (read about our exploration tools **here** or something)